src/Arcus.Observability.Telemetry.Core/Extensions/ILoggerExtensions.cs

```diff
@@ -189,6 +189,402 @@ public static class ILoggerExtensions
+        /// <param name="context">The telemetry context that provides more insights on the Azure Service Bus topic request.</param>
+        /// <exception cref="ArgumentNullException">Thrown when the <paramref name="logger"/> or the <paramref name="measurement"/> is <c>null</c>.</exception>
+        /// <exception cref="ArgumentException">Thrown when the <paramref name="serviceBusNamespacePrefix"/> or <paramref name="topicName"/> is blank.</exception>
+        public static void LogServiceBusTopicRequestWithPrefix(
+            this ILogger logger,
+            string serviceBusNamespacePrefix,
+            string topicName,
+            string operationName,
+            bool isSuccessful,
+            DependencyMeasurement measurement,
+            Dictionary<string, object> context = null)
+        {
+            Guard.NotNull(logger, nameof(logger), "Requires an logger instance to track telemetry");
+            Guard.NotNullOrWhitespace(serviceBusNamespacePrefix, nameof(serviceBusNamespacePrefix), "Requires an Azure Service Bus namespace to track the topic request");
+            Guard.NotNullOrWhitespace(topicName, nameof(topicName), "Requires an Azure Service Bus topic name to track the topic request");
+            Guard.NotNull(measurement, nameof(measurement), "Requires an instance to measure the Azure Service Bus topic request process latency duration");
+
+            // TODO: the dependency date from the measurement instance is not used when tracking the request.
+
+            LogServiceBusTopicRequestWithPrefix(logger, serviceBusNamespacePrefix, topicName, operationName, isSuccessful, measurement.Elapsed, measurement.StartTime, context);
+        }
+
+        /// <summary>
+        ///     Logs an Azure Service Bus topic request.
+        /// </summary>
+        /// <param name="logger">The logger instance to track the telemetry.</param>
+        /// <param name="serviceBusNamespace">The namespace where the Azure Service Bus topic is registered.</param>
+        /// <param name="topicName">The name of the Azure Service Bus topic.</param>
+        /// <param name="operationName">The name of the operation of the request.</param>
+        /// <param name="isSuccessful">The indication whether or not the Azure Service Bus topic request was successfully processed.</param>
+        /// <param name="measurement">The instance to measure the latency duration of the Azure Service Bus topic request.</param>
+        /// <param name="context">The telemetry context that provides more insights on the Azure Service Bus topic request.</param>
+        /// <exception cref="ArgumentNullException">Thrown when the <paramref name="logger"/> or the <paramref name="measurement"/> is <c>null</c>.</exception>
+        /// <exception cref="ArgumentException">Thrown when the <paramref name="serviceBusNamespace"/> or <paramref name="topicName"/> is blank.</exception>
+        public static void LogServiceBusTopicRequest(
```

Comment:
So, related to my previous comment, this method would be named `LogServiceBusTopicRequestWithSuffix` and this method introduces an extra parameter where you can specify the 'cloud' that is used.

It would then be called like this:

```
logger.LogServiceBusTopicRequestWithSuffix ("mynamespacename", suffix: ".servicebus.cloudapi.de", topicName: "mytopic", .... );
```

Reply:
Normally, this is already available in the Azure SDK, but it seems unnecessary to reference this. So yeah, I think the best way in between is to use the `string` representation here. People can still reference the Azure SDK themselves, then.

Reply:
I agree.  It would be nice if we could just mention this in the documentation.

Reply:
Created an issue for this. #267